assets/css/home.css

```diff
@@ -1,4 +1,4 @@
-element.style {
+.container {
```

Comment:
Does this class still exist in your HTML? Double-check that it does what you want it to do.

Reply:
It no longer exists in the html.  I removed it from the css file as well.